src/App.js

```diff
@@ -0,0 +1,27 @@
+//import CreateNewRestaurant from './forms/CreateNewRestaurant';
+import React, {useState, useRef} from 'react'; 
+//import DropDownNew from './dropdown';
+import UserInfo from './Components/Users';
+import AdminInfo from './Components/Admin';
```

Comment:
Try to be more accurate - User or UserInfo? if the last, the users file should be the same